### DISCUSSION TITLE
无法使用邮箱接收验证码登录会员。环境 Win11，Lantern Beta版本，可能是7.8.8，下载日期2024-06-20

### DISCUSSION THREAD

**Discussion by daxiong0513** [getlantern/lantern-forum-cn#624] [Q&A]
使用注册和充值过的邮箱，无法接收验证码。环境 Win11，Lantern Beta版本，可能是7.8.8，下载日期2024-06-20

Mac OS也是同样的问题。

之前2024-4，提过一次充错会员账号的帖子。链接如下，里面有充值凭证和账号信息。
https://github.com/getlantern/lantern-forum-cn/discussions/352#discussioncomment-9145628

**Comment by MaxMKMax** [comment]
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
@daxiong0513 您好. 为了保护您的隐私. 请您发支付单据和账号又想到这个链接. 谢谢. 
您发完后请自行删掉个您的个人资料和单据.
